docs/running/server/wes.rst

```diff
@@ -79,28 +79,48 @@ Below is a detailed summary of all available options:
 .. _GA4GH docs on CORS: https://w3id.org/ga4gh/product-approval-support/cors
 
 
+.. _WESRunWithDockerCompose:
+
 Running the Server with `docker-compose`
 ----------------------------------------
 
 Instead of manually setting up the server components (``toil server``, RabbitMQ, and Celery), you can use the following
 ``docker-compose.yml`` file to orchestrate and link them together.
 
-Make sure to change ``/tmp/toil-workflows`` if you want Toil workflows to live somewhere else, and create the directory
+Make sure to change the credentials for basic authentication by updating the
+``traefik.http.middlewares.auth.basicauth.users`` label. (Note that single ``$`` signs need to be replaced with ``$$``
+in the yaml file).
+
+When running on a different host other than ``localhost``, make sure to change the ``Host`` to your
+tartget host in the ``traefik.http.routers.wes.rule`` and ``traefik.http.routers.wespublic.rule`` labels.
+
+You can also change ``/tmp/toil-workflows`` if you want Toil workflows to live somewhere else, and create the directory
 before starting the server.
 
-Also make sure to run it behind a firewall; it opens up the Toil server on port 8080 to anyone who connects.
+In order to run workflows that require Docker, the ``docker.sock`` socket must be mounted as volume for Celery.
+Additionally, the ``TOIL_WORKDIR`` directory (defaults to: ``/var/lib/toil``) and ``/var/lib/cwl`` (if running CWL
+workflows with ``DockerRequirement``) should exist on the host and also be mounted as volumes.
 
-.. literalinclude:: ./Dockerfile
-   :language: yaml
+
+Also make sure to run it behind a firewall; it opens up the Toil server on port 8080 to anyone who connects.
 
 .. literalinclude:: ./docker-compose.yml
    :language: yaml
-   :emphasize-lines: 15,25,30
+   :emphasize-lines: 13-15, 22,25, 30,33,34, 43,45
+
+
+Further customization can also be made as needed. For example, if you have a domain, you can set up HTTPS with Let's
+Encrypt `as such`_.
```

Comment:
I would say "like this", or just make the text of the link the thing that they are doing ("set up HTTPS with Let's Encrypt").